platforms/core-runtime/base-asm/src/main/java/org/gradle/model/internal/asm/BytecodeFragment.java

```diff
@@ -20,7 +20,11 @@
 
 public interface BytecodeFragment {
 
-    BytecodeFragment NO_OP = visitor -> {};
+    @SuppressWarnings("Convert2Lambda")
```

Comment:
❓ Why is this needed if the code is Java 6 only?

Reply:
Yes, not needed, fixed